src/components/Utils/MobileFormat.js

```diff
@@ -0,0 +1,18 @@
+export const validateMobileFormat = (string) => {
+
+  // initialize local variables
+  var result = true;
+  var mobile_pattern = /^[\d\s]{10,12}$/; 
```

Comment:
I can do `045 487945` and still pass, but that's an invalid mobile number. I think you should sanitize the string to be clear of spaces first before running the regex and just check for 10 digits